src/unity/extensions/CMakeLists.txt

```diff
@@ -47,4 +48,11 @@ make_extension(dictionary_transformation_extensions SOURCES
 # )
 # target_link_libraries(timeseries PUBLIC grouped_sframe)
 
+#----------------------------
```

Comment:
Where did this come from? Doesn't look related to this change.

Reply:
Good catch.